### PR TITLE
feat: 1991 /parties integration tests

### PR DESCRIPTION
**Pull request by eoln**

### CODE REVIEW

**Comment by kleyow** [comment]
@eoln rebase/merge this off master. diff contains the async2sync work.

File: docker-compose.integration.yml [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
+
+networks:
+  mojaloop-net:
+    name: mojaloop-net
+
+#volumes:
+#  node_modules:
 
 services:
-  redis:
```
Comment (kleyow) [inline docker-compose.integration.yml:6]: Hmm...how is this working when removed when the scheme adapter depends on it?

**Review by kleyow — DISMISSED** [review]
looks good just one question.

File: docker-compose.integration.yml (hunk 1 above)
Comment (eoln) [reply docker-compose.integration.yml:6]: this file is deleted

**Comment by lewisdaly** [comment]
I broke the Pr title check :(

File: docker/ml-testing-toolkit/spec_files/reports/templates/newman/script.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
+function now () {
```
Comment (lewisdaly) [inline docker/ml-testing-toolkit/spec_files/reports/templates/newman/script.js:1]: hmm looks like a lot of copy pasting...

**Review by lewisdaly — APPROVED** [review]

**Review by lewisdaly — APPROVED** [review]

**Pull request merged**